src/test/runTest.ts

```diff
@@ -40,7 +40,7 @@ async function main() {
 
 		const vscodeExecutablePath = await downloadAndUnzipVSCode('stable');
 		const cliPath = resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath, "linux-x64");
-		cp.spawnSync(cliPath, ['--install-extension', 'ms-python.python'], {
+		cp.spawnSync(cliPath, ['--install-extension', 'ms-python.python', '--force'], {
```

Comment:
It doesn't work without this anymore?

Reply:
This flag makes sure that we always check whether there is a newer version of the extensions available and upgrade to it if there is one. It should just make sure that we get faster feedback when there is a breaking change. So to answer your question: without this flag the tests should also work.